crates/rooch-config/src/lib.rs

```diff
@@ -82,9 +82,28 @@ pub struct RoochOpt {
     /// The Ethereum RPC URL to connect to for relay L1 block and transaction to L2.
     /// If not set, the relayer service will not start.
     #[serde(skip_serializing_if = "Option::is_none")]
-    #[clap(long)]
+    #[clap(long, env = "ETH_RPC_URL")]
     pub eth_rpc_url: Option<String>,
 
+    /// The Bitcoin RPC URL to connect to for relay L1 block and transaction to L2.
+    /// If not set, the relayer service will not start.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[clap(
+        long,
+        env = "BITCOIN_RPC_URL",
```

Comment:
Is env configured through `.env` file?

Reply:
From the env variable. Some users do not wants expose the password to the cli command argument, can use the env.